python/pip_install/tools/dependency_resolver/dependency_resolver.py

```diff
@@ -88,6 +88,10 @@ def _select_golden_requirements_file(
     requirements_in_path = Path(requirements_in)
     resolved_requirements_in = str(requirements_in_path.resolve())
 
+    requirements_file = _select_golden_requirements_file(
```

Comment:
Thanks for adding this functionality. This allows users to not need to run the `[[uname -r == foo ]] && cp requirements.txt requirements_foo.txt` themselves.